macOS Monterey uses Port 7000

## Describe the bug

The `fluid.testem.base` grade uses Port 7000 as the default for the [`coveragePort`](https://github.com/fluid-project/fluid-testem/blob/9f909808ad3aba9acc3dfc43102a372f67436103/src/js/testem-component.js#L364). In macOS 12 (Monterey) that port is used by Control Centre for the AirPlay feature ( see: https://developer.apple.com/forums/thread/682332 ). This is causing the test to error out because the port is already in use.

```log
08:11:21.736:  FATAL ERROR: Uncaught exception: listen EADDRINUSE: address already in use :::7000
Error: listen EADDRINUSE: address already in use :::7000
    at Server.setupListenHandle [as _listen2] (net.js:1318:16)
    at listenInCluster (net.js:1366:12)
    at Server.listen (net.js:1452:7)
    at Function.listen (/Users/jo/Documents/git/infusion/node_modules/express/lib/application.js:618:24)
    at fluid.express.startServer (/Users/jo/Documents/git/infusion/node_modules/fluid-express/src/js/express.js:48:32)
    at togo (/Users/jo/Documents/git/infusion/src/framework/core/js/FluidIoC.js:2844:33)
    at fluid.event.firer.fire (/Users/jo/Documents/git/infusion/src/framework/core/js/Fluid.js:1775:40)
    at fluid.express.routable.connectDirectDescendants (/Users/jo/Documents/git/infusion/node_modules/fluid-express/src/js/routable.js:70:37)
    at togo (/Users/jo/Documents/git/infusion/src/framework/core/js/FluidIoC.js:2844:33)
    at fluid.event.firer.fire (/Users/jo/Documents/git/infusion/src/framework/core/js/Fluid.js:1775:40)
    at fluid.express.init (/Users/jo/Documents/git/infusion/node_modules/fluid-express/src/js/express.js:43:43)
    at togo (/Users/jo/Documents/git/infusion/src/framework/core/js/FluidIoC.js:2844:33)
    at fluid.event.firer.fire (/Users/jo/Documents/git/infusion/src/framework/core/js/Fluid.js:1775:40)
    at fluid.concludeComponentInit (/Users/jo/Documents/git/infusion/src/framework/core/js/FluidIoC.js:1532:30)
    at /Users/jo/Documents/git/infusion/src/framework/core/js/FluidIoC.js:2203:29
    at Array.forEach (<anonymous>)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! infusion@4.0.0 test:browser: `testem ci --file tests/testem.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the infusion@4.0.0 test:browser script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/jo/.npm/_logs/2021-10-27T12_11_21_756Z-debug.log
ERROR: "test:browser" exited with 1.
npm ERR! Test failed.  See above for more details.
```
## Technical details

### Desktop
 - OS: macOS 12 (Monterey)

--- a/docs/testem-component.md
+++ b/docs/testem-component.md
@@ -18,7 +18,7 @@ see below.
 | `cwd`                     | `{String}`  | Defaults to `process.cwd()`, i.e. the directory from which the script was called. |
 | `testemDir`               | `{String}`  | The directory in which testem's browser settings and temporary files should be stored. |
 | `wrappedEventTimeout`     | `{Number}`  | How long to wait (in milliseconds) before triggering a timeout when waiting for key startup/shutdown events. |
-| `coveragePort` (required) | `{Number}`  | The port fluid-express should listen on to record coverage data. Defaults to `7000`.|
+| `coveragePort` (required) | `{Number}`  | The port fluid-express should listen on to record coverage data. Defaults to `7003`.|
 | `coverageUrl`             | `{String}`  | The URL on which the coverage listener should be run.  Set based on `coveragePort` by default. |
 | `coverageDir` (required)  | `{String}`  | The full or package-relative path where coverage data should be saved. By default, a unique subdirectory is created in `os.tmpdir()`. |
 | `reportsDir` (required)   | `{String}`  | The full or package-relative path where coverage reports and test results should be saved. By default, a unique subdirectory is created in `os.tmpdir()`. |

--- a/src/js/coverageClientMiddleware.js
+++ b/src/js/coverageClientMiddleware.js
@@ -32,7 +32,7 @@ fluid.defaults("fluid.testem.middleware.coverageClient", {
     path:    "/client",
     baseClientSource: "%fluid-testem/src/js/client/coverageSender.js",
     clientInvokerTemplatePath: "%fluid-testem/src/templates/coverage-client-invoker.handlebars" ,
-    coveragePort: 7000,
+    coveragePort: 7003,
     hookTestem: true,
     hookQUnit: false,
     exposeCallback: false,

--- a/src/js/testem-component.js
+++ b/src/js/testem-component.js
@@ -361,7 +361,7 @@ fluid.testem.constructProxies = function (sourceDirs, contentDirs, additionalPro
 
 fluid.defaults("fluid.testem.base", {
     gradeNames:  ["fluid.component"],
-    coveragePort: 7000,
+    coveragePort: 7003,
     coverageUrl: {
         expander: {
             funcName: "fluid.stringTemplate",

